iabtcf-decoder/src/main/java/com/iabtcf/decoder/TCStringV2.java

```diff
@@ -480,11 +481,60 @@ public IntIterable getCustomPurposesLITransparency() {
         if (cache.add(PPTC_CUSTOM_PURPOSES_LI_TRANSPARENCY)) {
             customPurposesLITransparency = BitSetIntIterable.EMPTY;
 
-            ByteBitVector dvBbv = getSegment(SegmentType.PUBLISHER_TC);
+            BitReader dvBbv = getSegment(SegmentType.PUBLISHER_TC);
             if (dvBbv != null) {
                 customPurposesLITransparency = fillBitSet(dvBbv, PPTC_CUSTOM_PURPOSES_LI_TRANSPARENCY);
             }
         }
         return customPurposesLITransparency;
     }
+
+    @Override
```

Comment:
how about introducing lombok to reduce boilerplate code?

Reply:
no

Reply:
why?